services/brig/test/integration/API/Search.hs

```diff
@@ -452,7 +456,7 @@ withOldIndex opts f = do
   indexName <- randomHandle
   createIndexWithMapping opts indexName oldMapping
   let newOpts = opts & Opt.elasticsearchL . Opt.indexL .~ indexName
-  withSettingsOverrides newOpts f <* deleteIndex opts indexName
+  withSettingsOverrides newOpts f
```

Comment:
what's this about?

Reply:
left-over from debuginng and accidentally commited. I reverted it